### PR TITLE
Issues 428

### PR DESCRIPTION
**Pull request by enriquesouza**
This fixes all issues from https://github.com/vultisig/vultisig-ios/issues/428

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Keysign/KeysignDiscoveryView.swift [hunk 1]
```diff
@@ -169,6 +169,7 @@ struct KeysignDiscoveryView: View {
                 .onAppear {
                     if participantDiscovery.peersFound.count == 1 && participantDiscovery.peersFound.first == peer {
                         handleSelection(peer)
+                        startKeysign()
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Keysign/KeysignDiscoveryView.swift:172]: For this one , we need to make sure that we already have more than the threshold before we kick off keysign. 

can you make sure `viewModel.selections.count < (vault.getThreshold() + 1)` meet?

Comment (enriquesouza) [reply VultisigApp/VultisigApp/Views/Keysign/KeysignDiscoveryView.swift:172]: Hum... Let me double-check.

Comment (enriquesouza) [reply VultisigApp/VultisigApp/Views/Keysign/KeysignDiscoveryView.swift:172]: Done

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**